tests/suites/test_suite_bignum_mod_raw.function

```diff
@@ -520,7 +520,59 @@ exit:
 /* END MERGE SLOT 5 */
 
 /* BEGIN MERGE SLOT 6 */
+/* BEGIN_CASE */
+void mpi_mod_raw_canonical_to_modulus_rep( const char *input_N, int rep,
+                                           const char *input_A,
+                                           const char *input_X )
+{
+    mbedtls_mpi_mod_modulus N;
+    mbedtls_mpi_mod_modulus_init( &N );
```

Comment:
I know why you do this (have initialisation as close as possible to where the variable is declared) but actually I find it gets hidden in the rest of the variable declarations.

What I've found much easier for me to read is

```
mbedtls_mpi_uint *A = NULL;
mbedtls_mpi_uint *A = NULL;
size_t A_limbs, X_limbs = 0;

mbedtls_mpi_mod_modulus N;
mbedtls_mpi_mod_modulus_init( &N );
```

so that the modulus is the last of the variable declarations, with a blank line after the other ones.

Reply:
I don't see the problem here. It's perfectly fine that it gets hidden: it's just initialization, the only useful information is that it's present. It's only a problem if this is fully missing (which Msan will tell us, if nothing else), or too late (which we might only find out when there's a test failure, if it's after a point where the tests can fail). So putting the initialization next to the declaration makes it easier to spot mistakes, both manually and via tools.